src/modules/auth/controllers/auth.controller.ts

```diff
@@ -139,4 +142,22 @@ export class AuthController {
       return { error: err.message }; // this needs to be updated with a more generic approach
     }
   }
+  // will be removed after testing the contract flow!!!
```

Comment:
Keeping this intentional? Or should this comment be removed?

Reply:
Need to clean-up this flow, as suggested above as well. The calls should be made from API-Gatewat and Fabric Gateway should only be receiving these and calling the contract.
These comments were added to remind that it's only the the sole purpose of testing the call to the contract.